WEBPACK_UPDATE.md

```diff
@@ -0,0 +1,59 @@
+# Upgrading from angular-cli@1.0.0-beta.10 to the angular-cli@1.0.0-beta.11-webpack 
+
+To update `angular-cli` to the webpack preview, you must update both the global package, project's local package and your project files.
+
+## Global package:
+```
+npm uninstall -g angular-cli
+npm cache clean
+npm install -g angular-cli@1.0.0-beta.11-webpack
+```
+
+## Local project package:
+```
+rm -rf node_modules dist tmp typings
+npm install --save-dev angular-cli@1.0.0-beta.11-webpack
+```
+
+IMPORTANT NOTE: 
+Currently project generated with `ng new` will use a wrong CLI version (see https://github.com/angular/angular-cli/issues/1528). After initializing your project, run `npm install --save-dev angular-cli@1.0.0-beta.11-webpack` to set the correct version.
+
+## Project files
+
+You will need to run `ng init` to check for changes in all the auto-generated files created by `ng new` and allow you to update yours. You are offered four choices for each changed file: `y` (overwrite), `n` (don't overwrite), `d` (show diff between your file and the updated file) and `h` (help).
+
+Carefully read the diffs for each code file, and either accept the changes or incorporate them manually after `ng init` finishes.
+
+You can find a sample github diff of the changes introduced between a beta.10 and webpack preview project at https://github.com/filipesilva/angular-cli-webpack-upgrade/commit/HEAD.
+
+Here is a summary of the file changes:
```

Comment:
Separate this into 5 groups: Unit tests, E2E tests, App code / Components, Project Config, and App Config.

Example:

----

# Unit Tests

* Replaced `karma-test-shim.js` with `test.ts` file in the main entry. The content of that file can be found here: https://github.com/angular/angular-cli/blob/master/addon/ng2/blueprints/ng2/files/__path__/test.ts

### Karma Configuration
* Introduced an `angular-cli` karma plugin to help us integrate with karma. To add this plugin, replace your `frameworks` and `plugins` entries in your karma config by these:  
```
    frameworks: ['jasmine', 'angular-cli'],
    plugins: [
      require('karma-jasmine'),
      require('karma-chrome-launcher'),
      require('karma-coverage'),
      require('angular-cli/plugins/karma')
    ],
```

* Note that we also added `karma-coverage` to handle coverage reporting. If you don't want coverage, you can safely remove it.
* Removed all the `files` entries. We only need to link the entry point for testing, which is `test.ts`. Replace your files entry by:
```
    files: [
      { pattern: './src/test.ts', watched: false }
    ],
```

* We need to add a preprocessor for the file, as it will be handled by the `angular-cli` plugin:
```
    preprocessors: {
      './src/test.ts': ['angular-cli']
    },
```